QUEST.tsv

```diff
@@ -773,101 +773,101 @@ QUEST_20150317_000772	There are some old pots among the new pots. {nl}Which one
+QUEST_20150317_000823	It is unknown who the statue honors or how old it is. {nl}But it certainly has mysterious powers that move selected people to other areas.
 QUEST_20150317_000824	Good job. {nl}I have done my task. {nl}It is my time now.
-QUEST_20150317_000825	Go forward. {nl}The Sculptor is waiting for you. {nl}I was glad to have guided you. 
+QUEST_20150317_000825	Go forward. {nl}The Sculptor is waiting for you. {nl}I was glad to have guided you.
 QUEST_20150317_000826	Ah…
-QUEST_20150317_000827	Danger! The vacant lot below these crossroads {nl} has regularly seen dangerous monsters. Entrance is forbidden! {Nl} - The Fedimian Market
-QUEST_20150317_000828	People are now praying less and less to the Goddesses. {Nl} Maybe they are losing hope…
-QUEST_20150317_000829	How can I forget you… {nl}Please do not roam around this cursed land {nl}and go to the arms of the Goddess.
+QUEST_20150317_000827	Danger! The vacant lot below these crossroads {nl}has regularly seen dangerous monsters. Entrance is forbidden! {nl} - The Fedimian Market
+QUEST_20150317_000828	People are now praying less and less to the Goddesses. {nl}They're losing hope…
+QUEST_20150317_000829	How could I forget you… {nl}Please do not roam around this cursed land {nl}and go to the arms of the Goddess.
 QUEST_20150317_000830	Holiday Notice
-QUEST_20150317_000831	The owner has been enlisted to the army to save the Kingdom. {nl}In the meantime, the restaurant will be closed until further notice.{nl}- Owner of Klaipeda's specialty restaurant Vulinya
+QUEST_20150317_000831	I have been enlisted to the army to save the kingdom. {nl}Therefore, the restaurant will be closed in the meantime. {nl} - Owner of Klaipeda's Specialty Restaurant, Vulinya
 QUEST_20150317_000832	Don't blow out the candle. {nl}It guides the souls so they don't lose their way.
-QUEST_20150317_000833	As you know, the King died on Medis Diena and the fate of other royalty and nobility was no different. {Nl} The kingdom is holding together somehow but… now we can only pray.
+QUEST_20150317_000833	As you know, the King died on Medis Diena and the fate of other royalty and nobility were no different. {nl}The kingdom is holding together somehow but… now we can only pray.
 QUEST_20150317_000834	Now we can only pray…
-QUEST_20150317_000835	When the spire is bathed with a red light the… door… to fantasy… shall be opened.
+QUEST_20150317_000835	When the spire is bathed with a red light the… door… to fantasy shall be opened.
 QUEST_20150317_000836	Will of King Zachariel
-QUEST_20150317_000837	Dear Revelator.{nl}Listen to the interpretations of the revelation.
-QUEST_20150317_000838	Huh…{nl}Who would've thought I'd be swayed by humans…
-QUEST_20150317_000839	I shall kill you in the most painful way!
+QUEST_20150317_000837	Dear Revelator. {nl}Listen to the interpretations of the revelation.
+QUEST_20150317_000838	Huh… {nl}Who'd've thought I'd be swayed by humans…
```

Comment:
I have to question changing "Who would've" to "Who'd've" here.  It's typically only appropriate if you're quoting someone and reproducing colloquial speech.